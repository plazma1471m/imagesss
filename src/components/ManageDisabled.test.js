import React from 'react'
import ManageDisabled from './ManageDisabled'
import renderer from 'react-test-renderer'

it('renders disabled sites management correctly', () => {
  const tree = renderer.create(<ManageDisabled />).toJSON()
  expect(tree).toMatchSnapshot()
})
