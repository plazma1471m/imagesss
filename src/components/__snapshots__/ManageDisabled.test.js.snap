// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders disabled sites management correctly 1`] = `
<form
  action={undefined}
  className="ui form"
  onSubmit={[Function]}
>
  <div
    className="field"
  >
    <textarea
      disabled={undefined}
      onChange={[Function]}
      required={undefined}
      rows={4}
      style={
        Object {
          "resize": "",
        }
      }
      type={undefined}
      value=""
    />
  </div>
</form>
`;
